apiserver/facades/client/applicationoffers/applicationoffers_test.go

```diff
@@ -104,7 +107,7 @@ func (s *applicationOffersSuite) assertOffer(c *gc.C, expectedErr error) {
 	}
 
 	if expectedErr == nil {
-		modelUUID := model.UUID(testing.ModelTag.Id())
+		modelUUID := coremodel.UUID(testing.ModelTag.Id())
```

Comment:
modelUUID := modeltesting.GenModelUUID(c)

Reply:
Changed all to use moelUUID from modeltesting.GenModelUUID(c)